site/how-it-works/encryption.md

```diff
@@ -32,6 +33,11 @@ extensions to TLS such as [DTLS](https://tools.ietf.org/html/rfc4347)
 which can operate over UDP, these are not widely implemented and
 deployed.
 
+In the case of fast datapath, UDP packets are encrypted by using
```

Comment:
I would put this section ahead of Sleeve, because it will be the default mode for most people.